test_data/expression_test_data/functions_trigonometry.json

```diff
@@ -225,6 +225,34 @@
         "value": 0.0
       },
       "platforms": [
+        "android"
+      ]
+    },
+    {
+      "expression": "@{asin(-1.0)}",
```

Comment:
There are other tests for `asin` below. This case seems to be unnecessary because case with parameter -0.5 already exists.
You should add `android` to platforms in those cases.

Reply:
Delete this test asin(-1.0) and add "android" to test asin(-0.5)